cmd/translate/main.go

```diff
@@ -57,10 +71,27 @@ var rootCmd = &cobra.Command{
 		defer grpcServer.GracefulStop()
 
 		mux := runtime.NewServeMux()
-		translatev1.RegisterTranslateServiceServer(grpcServer, &translate.TranslateServiceServer{})
+
+		var (
+			repository repo.Repo
+			errRepo    error
```

Comment:
Advisory. In this case, you don't have to declare a new error instance. You can reuse `err`.